octodns/record/__init__.py

```diff
@@ -183,15 +183,11 @@ def excluded(self):
     def included(self):
         return self._octodns.get('included', [])
 
-    @property
-    def healthcheck_host(self):
+    def healthcheck_host(self, value=None):
         healthcheck = self._octodns.get('healthcheck', {})
         if healthcheck.get('protocol', None) == 'TCP':
             return None
-        try:
-            return healthcheck['host']
-        except KeyError:
-            return self.fqdn[:-1]
+        return healthcheck.get('host', self.fqdn[:-1]) or value
```

Comment:
It would use `value` if `healthcheck['host']` is false irrespective of `self.fqdn[:-1]` . It is basically the same logic as it was before with try and except. This one liner can also be:

```
        try:
            return healthcheck['host'] or value
        except KeyError:
            return self.fqdn[:-1]
```

Reply:
Ah. I see the path you were aiming for, `host` exists and is false-y. Makes sense.

Reply:
Yup. So is this PR good to merge? :)

Reply:
Probably. Going to look over it later this evening when I get a chance to focus.